src/sentry/tagstore/snuba/backend.py

```diff
@@ -391,3 +413,7 @@ def get_group_ids_for_search_filter(
         # exist in Snuba. This logic is implemented in the search backend
         # instead.
         raise NotImplementedError
+
+    def update_group_for_events(self, project_id, event_ids, destination_id):
+        # Group updates are unncessary in Snuba, but we shouldn't throw an error.
+        pass
```

Comment:
This seems unrelated to the rest of this change. Was there some reason to include this here rather than making this change separately?

Reply:
Errr nope, those were all just stubs from my scan of what-methods-we-need-to-add. I can move it out.